lib/fastlane/actions/create_keychain.rb

```diff
@@ -28,6 +28,15 @@ def self.run(params)
         command << " ~/Library/Keychains/#{escaped_name}"
 
         commands << Fastlane::Actions.sh(command, log: false)
+
+        if params[:add_to_search_list]
+          keychains = []
+          keychains += Fastlane::Actions.sh("security list-keychains -d user", log: false).shellsplit unless Helper.test?
```

Comment:
When running test, the `sh` action just echoes the command instead of returning the keychain search list, which means the final command would look like that:
```
security list-keychains -s security list-keychains -d user /Users/myself/Library/Keychains/test.keychain
```
If course, this command wouldn't be executed either. But I thought it would be odd to write a test that expects the action to output incorrect shell commands.